<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="FasTab">
       <attribute name="title">
        <string>火警监测</string>
       </attribute>
       <widget class="QComboBox" name="fasList">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>301</width>
          <height>31</height>
         </rect>
        </property>
        <property name="maxVisibleItems">
         <number>50</number>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpen">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpen_2">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>10</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnOpen_3">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>写入</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="ConfigTab">
       <attribute name="title">
        <string>配置</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>212</height>
         </rect>
        </property>
        <property name="title">
         <string>串口配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>端口号:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_serialNum">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM8</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM9</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>COM10</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>波特率:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_BaudRate">
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>230400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>460800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>576000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>921600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1152000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1500000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2000000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2500000</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>校验位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_Parity">
               <item>
                <property name="text">
                 <string>无</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>奇</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>偶</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>数据位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_ByteSize">
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>停止位:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_StopBit">
               <item>
                <property name="text">
                 <string>1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1.5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>185</width>
          <height>140</height>
         </rect>
        </property>
        <property name="title">
         <string>联动配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>联动编号:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>联动串口:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QSpinBox" name="spinBox_5"/>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_2"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QCheckBox" name="checkBox_4">
               <property name="text">
                <string>启动联动</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>231</width>
          <height>140</height>
         </rect>
        </property>
        <property name="title">
         <string>电源板配置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>电源板串口:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>关机电压(毫伏):</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QSpinBox" name="spinBox_6"/>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox_3"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QCheckBox" name="checkBox_5">
               <property name="text">
                <string>启动</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btn_gatewayServerSet">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>网关服务器配置</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
